README.md

```diff
@@ -1,18 +1,22 @@
 ﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | @nasaku898
-* Émilie Martin | @emilie-martin
-* Daniela Venuta | @daniela-venuta
-* Minh-Tam Do | @mint47
-* Luc Nguyen | @1999lucnguyen
-* Karin Kazarian | @Karin-kazarian
+Simon Lim (tech lead) | @nasaku898
+:--|:--
+Émilie Martin         | @emilie-martin  
+Daniela Venuta        | @daniela-venuta
+Minh-Tam Do           | @mint47
+Luc Nguyen            | @1999lucnguyen  
+Karin Kazarian        | @Karin-kazarian  
 
 ### Description
-This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+This is a social media project based on the main features of instagram. This project is an introduction that we created to help us learn  the basics of industrial software development process.
+The project was created for SOEN341 at Concordia University.
+
 ### Objectives
-The app is a replication of the social media Instagram from which we will add on features.
+The app is intended to implement the core features of Instagram, such that the user is able to post a picture, follow another user, and be allowed to comment under posted pictures. The project guideline also requires us to implement additional features.
```

Comment:
Don't know if we need to mention the fact we're gonna implement additional features...? Think it'd be sufficient to simply add them once we discuss them with the TAs at the end of Sprint 3

Reply:
for now will probably leave as is and change it afterwards